pytest_rpc/helpers.py

```diff
@@ -216,10 +217,162 @@ def delete_instance(instance_name, run_on_host):
+    sleep(10)
+
+
+def verify_asset_in_list(service_type, service_name, run_on_host):
+    """Verify if a volume/server/image is existing
+
+    Args:
+        service_type (str): The OpenStack object type to query for.
+        service_name (str): The name of the OpenStack object to query for.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+
+    Returns:
+        boolean: Whether the expected asset was found or not.
+    """
+    cmd = "{} openstack {} list'".format(utility_container, service_type)
+    output = run_on_host.run(cmd)
+    if service_name in output.stdout:
+        return True
+    else:
+        return False
+
+
+def stop_server_instance(instance_name, run_on_host):
+    """Stop an OpenStack server/instance
+
+    Args:
+        instance_name (str): The name of the OpenStack instance to be stopped.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+    """
+    instance_id = get_id_by_name('server', instance_name, run_on_host)
+
+    cmd = "{} openstack server stop {}'".format(utility_container, instance_id)
+    run_on_host.run_expect([0], cmd)
+
+
+def create_snapshot_from_instance(snapshot_name, instance_name, run_on_host):
+    """Create snapshot on an instance
+
+    Args:
+        snapshot_name (str): The name of the OpenStack snapshot to be created.
+        instance_name (str): The name of the OpenStack instance from which the snapshot is created.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+    """
+    instance_id = get_id_by_name('server', instance_name, run_on_host)
+    cmd = "{} openstack server image create --name {} {}'".format(utility_container, snapshot_name, instance_id)
+
+    run_on_host.run_expect([0], cmd)
+    sleep(10)
+
+
+def delete_it(service_type, service_name, run_on_host):
+    """Delete an OpenStack object
+
+    Args:
+        service_type (str): The OpenStack object type to query for.
+        service_name (str): The name of the OpenStack object to query for.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+
+    Raises:
+        AssertionError: If operation is unsuccessful. """
+
+    sleep(15)
+    id = get_id_by_name(service_type, service_name, run_on_host)
+    cmd = "{} openstack {} delete {}'".format(utility_container, service_type, id)
+    run_on_host.run_expect([0], cmd)
+    sleep(20)
+
+    assert not verify_asset_in_list(service_type, service_name, run_on_host)
+
+
+def create_floating_ip(network_name, run_on_host):
+    """Create floating IP on a network
+
+    Args:
+        network_name (str): The name of the OpenStack network object on which the floating IP is created.
+        run_on_host (testinfra.Host): Testinfra host object to execute the action on.
+
+    Raises:
+        ValueError: If it has the right type but an inappropriate value.
+
+    Returns:
+        str: The newly created floating ip name
```

Comment:
It seems like returning `None` is meaningful in this function.  It should probably be documented.